### ISSUE TITLE
Notify inactive workers when there's a lot of work available

### ISSUE THREAD

**Issue by shosti** [rainforestapp/tester-chrome-extension#690]
It would be nice if the extension could optionally notify inactive workers when there's a lot of work, so they might consider connecting. Shouldn't be too hard to do...

@rainforestapp/tester-product sound sane.

**Comment by VeryDisappointedTester** [comment]
That would mean even less work for the testers that are active 10+ hours. Only fair distribution wolud be one based on testers stats and their activity.

**Comment by hunt40** [comment]
also lets pointed this fact there is a lot of testers then work,if people inactive that means they are busy or not available to do tests

**Comment by VeryDisappointedTester** [comment]
I feel like i'm in the Twilight Zone episode for the last 4-5 weeks.
Every single company in the world would reward workers that are active all day and do their job well.
But instead you keep pushing them away and want to help workers that are satisfied with 1-5 hits a day.
It's like you are reading the manual "What must NOT be done if you want to run a company"

**Comment by shosti** [comment]
2 points:

1. This would be strictly optional, so people who wanted to be left alone could turn it off
2. This would only activate if there are *more* jobs than active testers (i.e. there's work for everyone)

**Comment by JrRandy** [comment]
Lets get all the errors fixed before worrying about bringing even more people online to cause more errors, and make testing more painful.

**Comment by helloworld111gh** [comment]
Yeah I'd say no to this feature request, back in the day of polling I saw
"There are no virtual machines available" errors a lot.
When we have more tests AND more VM's than users then maybe add it in

**Comment by VeryDisappointedTester** [comment]
In the last 2-3 hours i've earned more than i earned entire day before that. If u take that away then nothing is left.
There is never enough work. Just keep sending hits.

As people above me said there are a lot of problems with hits. Last few hours almost every hit was very slow. I can't imagine what would be if there were ... lets say another 500+ testers.

**Comment by MarinaCec** [comment]
@shosti first wait rainforest to stabilize and stop having errors, wait at least one month without new updates, please ;/

**Comment by MarinaCec** [comment]
The extension already has noise to warn, who is inactive because  is busy or because they don't want to test

**Comment by cookin2** [comment]
He clearly said:  *This would only activate if there are _more_ jobs than active testers (i.e. there's work for everyone)"
He's a forward thinker!  Customers would get their runs completed fast and any workers available would be offered the work.

**Comment by VeryDisappointedTester** [comment]
This practically means that we will never again get two hits in a row. Every time when there are more hits that testers RF will invite inactive people. After a month with this way you will loose all testers that were active all day long. Quality of results will decline. Just watch and see. When you realise what did you done it will be too late.
Most of the people who are active like 1-2 hours a day dont really care that much about stats.. It is just some thing that brings some minor income.
You have 10000 active testers in the last 24 hours. How many of them are watching AMA or reporting problems? Few hundred at best. Just think about that.

**Comment by katiakoroliova** [comment]
I'm online 16h doing rainforest tests, it would be unfair to people like me

**Comment by VeryDisappointedTester** [comment]
I wolud ask all testers to think about something.
How many times did you report obvious bug but your bug count did not go up?
How many times every day? How many times in a month?
We all know that there are a lot of testers that dont care. Call them YES workers if you want.
What will happen if they become vast majority of testers?
What result will client get in 10 minutes?

**Comment by Electroglitch** [comment]
So many :-1:  What is this YouTube? :laughing: 
Joking aside , let's not get ahead of ourselves, first iron out all these errors and hope there will be more available work than available testers... :+1:

**Comment by cookin2** [comment]
@VeryDisappointedTester   You've made some very good points that I hadn't thought of, thank you :)

**Comment by Dolazim** [comment]
Can you add pop up when hit is open we click on him and sound will be stop?

**Comment by shosti** [comment]
Locking thread for now because it's veered way off topic. 

I think y'all are vastly overrating the effect this would have--it's just a convenience feature for people who want it. (I know we've had requests for this sort of thing before--if anyone has an alternative suggestion that would serve the same purpose, please open up a new issue.)
